### DISCUSSION TITLE
Pro version

### DISCUSSION THREAD

**Discussion by Hthrhurd** [d4rken-org/sdmaid-se#1174] [Q&A] [answered]
Hello, I paid for the first pro version you had.  Do I have to pay again for this version?  I really like your app.

**Comment by MarkyMarkDE** [comment, marked as answer]
> Hello, I paid for the first pro version you had. Do I have to pay again for this version? I really like your app.

This topic you find in the FAQ -  Yes, they are different apps, so u need pay for SDMaid SE again, migration is impossible.

**Comment by Nemix777** [comment]
That sucks for SDMaid Pro users who bought the app late in it's life (example: me), but recently upgraded their device to Android 13 or higher.

I think it would be fair SDMaid Pro users get some kind do of discount towards SDMaid SE, that's really the only thing that's holding me back from purchasing SE.

**Reply by MarkyMarkDE** [reply]
> That sucks for SDMaid Pro users who bought the app late in it's life (example: me), but recently upgraded their device to Android 13 or higher.
> 
> I think it would be fair SDMaid Pro users get some kind do of discount towards SDMaid SE, that's really the only thing that's holding me back from purchasing SE.

You can buy here a sponsorship for period or for lifetime, the amount is from 1 Buck to unlimited, much cheaper as in playstore.
BTW: SD-Maid-SE is really cheap for that you get, equal if you pay lifetime here over sponsorship or playstore for fixed amount!
The time "get all for free" is over.
-> https://github.com/sponsors/d4rken

What really sucks are apps, that offer lifetime with 50% discount, you buy this app and after that the dev removed this app from store upload new one (exactly the same app) but now only subscription based payments!
You get now for your paid app no more updates, and you can't install it new after that (e.g. new phone).
Your paid bucks are down the toilette!

<hr>

FYI: http://www.gnu.org/philosophy/free-sw.html
> Quote: "Free software" is a matter of liberty, not price. To understand the concept, you should think of "free" as in "free speech," not as in "free beer."
Free software is a matter of the users' freedom to run, copy, distribute, study, change and improve the software.

**Comment by Nemix777** [comment]
Thanks for the quick reply, 

I think I'll just stick with SDMaid 1 Pro version I have for now, since my phone running HyperOS is still on Android 13 and I prefer using SDMaid 1 Pro as oppose to the built-in Cleaner app.

When I upgrade my phone next year (which should come with Android 15), then I'll think about upgrading to SDMaid 2/SE Pro if the new phone does not have a Cleaner app or the built-in Cleaner app really sucks.

**Comment by cf100clunk** [comment]
Many of us prefer to deGoogle our Android devices but now SD Maid SE makes it impossible for us to upgrade because it specifies Google Play. Years ago I gladly paid for an SD Maid Pro unlocker but now that I want the new SE version's upgrade I am stuck.

Are there any plans or methods for us to upgrade SE without Google Play?

**Reply by MarkyMarkDE** [reply]
> Many of us prefer to deGoogle our Android devices but now SD Maid SE makes it impossible for us to upgrade because it specifies Google Play. Years ago I gladly paid for an SD Maid Pro unlocker but now that I want the new SE version's upgrade I am stuck.
> 
> Are there any plans or methods for us to upgrade SE without Google Play?

No, you need pay addiotional, via Playstore (fixed prices) or here for the VOSS Version (sponsorship darken; free of amount).
Please read FAQ to SDMaid-SE, this question is answerd there more detailed.

**Reply by cf100clunk** [reply]
The answer is not in the FAQ.

I will try to explain better. I have several de-Googled Android devices. Lots of people use alternatives like GrapheneOS, CalyxOS, LinageOS, NeoStore, Aurora Store, MicroG, etc. and love your SD Maid app like I do. I do not want to use Google Play Store for ANYTHING!

I installed SD Maid SE 1.1.2rc0 (10102000). CorpseFinder and SystemCleaner work fine. When I use AppCleaner and try to use the Delete control I get a window asking to Confirm Deletion, and when I select Delete I am taken to a page titled Get SD Maid SE Pro and I want to buy an upgrade but a window pops up saying Google Play is unavailable, which I already know because it is a de-Googled device.

I hope you understand what I am asking. Are there any plans or methods for us to buy an upgrade for SD Maid SE without having to use Google Play? I really, really hope so.

**Reply by d4rken** [reply]
You can download Google-free versions from GitHub, F-Droid or IzzyOnDroid and upgrade via voluntary donation through GitHub.

**Reply by cf100clunk** [reply]
Thank you, I have just sponsored.  :)

**Reply by MarkyMarkDE** [reply]
thank you, with your sponsorship you step on the right track, we must FOSS keep alive :wink:
